src/process.h

```diff
@@ -268,6 +268,19 @@ class Process : public ProcessListFromProcessGroup::Element,
     return run_timestamp_;
   }
 
+  void set_current_primitive_call(int primitive_module, int primitive_index) {
```

Comment:
You could also just store the bcp and decode it later. It is a little bit more complex, because you have to decode the module and primitive index at the time this fails, but it feels a bit more robust and prepared for other cases (long running intrinsics, perhaps?).

Reply:
done.